examples/ciao/README.md

```diff
@@ -42,7 +50,7 @@ to install the `qemu` package
 ### ansible-galaxy
 Install the required roles from ansible-galaxy
 
-    ansible-galaxy install -r requirements.yml
+    ansible-galaxy install --ignore-certs -r requirements.yml
```

Comment:
Right, It seems it was a temporary issue with their servers.
I dont need --ignore-certs anymore. so I'm removing this